tests/copp/test_copp.py

```diff
@@ -417,24 +436,25 @@ def _teardown_multi_asic_proxy(dut, creds, test_params, tbinfo):
         return
 
     logging.info("Removing iptables rules and disabling eth0 port forwarding")
-    http_proxy, https_proxy = copp_utils._get_http_and_https_proxy_ip(creds)
     dut.command("sudo sysctl net.ipv4.conf.eth0.forwarding=0")
-    # Delete IP Table rule for http and ptf nn_agent traffic.
-    mgmt_ip = dut.host.options["inventory_manager"].get_host(dut.hostname).vars["ansible_host"]
-    # Delete Rule to communicate to http/s proxy from namespace
-    dut.command("sudo iptables -t nat -D POSTROUTING -p tcp --dport 8080 -j SNAT --to-source {}".format(mgmt_ip))
-    dut.command("sudo ip -n {} rule delete from all to {} pref 1 lookup default".format(test_params.nn_target_namespace, http_proxy))
-    if http_proxy != https_proxy:
-        dut.command("sudo ip -n {} rule delete from all to {} pref 2 lookup default".format(test_params.nn_target_namespace, https_proxy))
+    if not test_params.swap_syncd:
+        http_proxy, https_proxy = copp_utils._get_http_and_https_proxy_ip(creds)
+        # Delete IP Table rule for http and ptf nn_agent traffic.
+        mgmt_ip = dut.host.options["inventory_manager"].get_host(dut.hostname).vars["ansible_host"]
+        # Delete Rule to communicate to http/s proxy from namespace
+        dut.command("sudo iptables -t nat -D POSTROUTING -p tcp --dport 8080 -j SNAT --to-source {}".format(mgmt_ip))
+        if http_proxy != https_proxy:
+            dut.command("sudo ip -n {} rule delete from all to {} pref 2 lookup default".format(test_params.nn_target_namespace, https_proxy))
     # Delete Rule to communicate to ptf nn agent client from namespace
     ns_ip = dut.shell("sudo ip -n {} -4 -o addr show eth0".format(test_params.nn_target_namespace) + " | awk '{print $4}' | cut -d'/' -f1")["stdout"]
     dut.command("sudo iptables -t nat -D PREROUTING -p tcp --dport 10900 -j DNAT --to-destination {}".format(ns_ip))
-    dut.command("sudo ip -n {} rule delete from {} to {} pref 3 lookup default".format(test_params.nn_target_namespace, ns_ip, tbinfo["ptf_ip"]))
 
 
 @pytest.fixture(scope="function", autouse=False)
 def backup_restore_config_db(duthosts, enum_rand_one_per_hwsku_frontend_hostname):
     duthost = duthosts[enum_rand_one_per_hwsku_frontend_hostname]
+    if duthost.is_multi_asic:
```

Comment:
Looks like this check/return here is causing few tests where we skipped for multi_asic -- to exit with the following error,

```
31/01/2023 21:34:51 __init__._fixture_generator_decorator    L0071 INFO   | -------------------- fixture backup_restore_config_db setup starts --------------------
31/01/2023 21:34:51 __init__._fixture_generator_decorator    L0078 ERROR  |
StopIteration()
Traceback (most recent call last):
  File "sonic-mgmt-int/tests/common/plugins/log_section_start/__init__.py", line 74, in _fixture_generator_decorator
    res = next(it)
StopIteration
31/01/2023 21:34:51 __init__._fixture_generator_decorator    L0079 INFO   | -------------------- fixture backup_restore_config_db setup ends --------------------
```

 maybe we can SKIP them via test_mark_conditions file?

Reply:
updated. `test_mark_conditions file` i don;t think we can do as we skip only for multi-asic dut which is run-time property.

Reply:
@judyjoseph  : updated to use `tests_mark_conditions.yaml`